src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public FigureDrawable getRandomFigure() {
```

Comment:
Don't use verbs for class/interface names. The interface name 'FigureDrawable' suggests an action rather than an ability or a property. Consider renaming it to something that represents an entity or a property, like 'DrawableFigure' if it's meant to represent figures that can be drawn.